packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -131,6 +131,18 @@ cluster.addAutoScalingGroup(autoScalingGroup);
 
 If you omit the property `vpc`, the construct will create a new VPC with two AZs.
 
+By default, all machine images will auto-update to the latest version
+on each deployment, causing a replacement of the instances in your AutoScalingGroup.
+If task draining is enabled, ECS will transparently reschedule tasks on to the new
+instances before terminating your old instances. If you have disabled task draining,
```

Comment:
not very familiar with ECS, but I'm confused about why disabled task draining would lead a user to choose to cache the AMI. can you clarify?

Reply:
Yes. Without task draining, when the instance is replaced (because of the AMI change), all tasks are just aborted, without giving them a chance to gracefully reschedule first.

Reply:
Gotcha, I'd add a brief explanation of this if possible.
> If you have disabled task draining, meaning tasks will be aborted if the machine image updates and causes replacement, you can pick ...